doc/ref_model/chapters/chapter05.rst

```diff
@@ -169,6 +169,41 @@ Tracing                                Jaeger
 
 **Table 5-7a:** Service examples.
 
+Platform Services - Load Balancer Requirements
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The table below specifies a set of requirements for the Load Balancer platform service.
+
+========== ===================================================================== ============================================================================
+Reference  Requirement                                                           Notes
+========== ===================================================================== ============================================================================
+pas.lb.001 The Load Balancer must support resource scaling	
+pas.lb.002 The Load Balancer must support resource resiliency	
+pas.lb.003 The Load Balancer must support scaling and resiliency in the local    Define local environment: within a subnet, tenant network, Availability 
+           environment                                                           Zone of a cloud, ...
+pas.lb.004 The Load Balancer should support resiliency in the global environment	
```

Comment:
What is the meaning of this in technical terms?

Reply:
@CsatariGergely Made a minor edit -- please review. Thanks

Reply:
Now it reads: `The Load Balancer should support resiliency in the  environment` I think it is either redundant to pas.lb.003 or needs more clarification.

Reply:
@CsatariGergely One is about resources and the other for the environment.